### ISSUE TITLE
attackAll unintended behaviour (probably)

### ISSUE THREAD

**Issue by Biviho** [miyo6032/MaelstromMod#30]
**Describe the bug**
with attackAll set to true, maelstrom mobs attach also eachother, don't know if this is intended or not

**To Reproduce**
-set attackAll to true
-watch them kill themselfs
-...
-profit

**Forge Version and Maelstrom Mod Version**
 forge: 14.23.5.2854
 mod: 1.12.2-0.4

**Comment by miyo6032** [comment]
Cannot replicate - do you have any other mods installed?

**Comment by Biviho** [comment]
yes, it's a pretty chonky modpack of 280 mods, sorry, i didn't tested it in a light environment.
mods that change mob behaviour in my modpack are epic siege mod and rough mobs revamped, i will test it in a few days
